### DISCUSSION TITLE
Multiple databases

### DISCUSSION THREAD

**Discussion by levu304** [blitz-js/blitz#3013] [Q&A] [answered]
I'm new on Blitz, i have a question
Can we connect multiple databases in a blitz project? If have, how can we do that?

**Comment by beerose** [comment, marked as answer]
Hey! It _should_ be possible. However, there is an issue that someone had problems with multiple Prisma clients: https://github.com/blitz-js/blitz/issues/2974, but you can find there some links with possible implementations.
